### PR TITLE
Synthetix support

### PR DESCRIPTION
**Pull request by marcelomorgado**

### CODE REVIEW

**Comment by adrianmcli** [comment]
wow @marcelomorgado great stuff again! I'll review this soon

File: tests/utils.ts [hunk 1]
```diff
@@ -2,3 +2,13 @@ import { ethers } from "ethers";
 import { BigNumber } from "ethers/utils";
 
 export const fromWei = (x: BigNumber, u = 18) => ethers.utils.formatUnits(x, u);
+
+export const mineBlock = async (provider: any, timestamp: number) =>
+  await provider.send("evm_mine", [timestamp]);
+
+export const increaseTime = async (provider: any, secsToIncrease: number) => {
```
Comment (adrianmcli) [inline tests/utils.ts:9]: This is really cool. We should probably publish this kind of stuff.

**Comment by adrianmcli** [comment]
@marcelomorgado looks like there's a conflict w/ `kyber.test.ts`. Do you think you can fix that?

**Comment by marcelomorgado** [comment]
> @marcelomorgado looks like there's a conflict w/ `kyber.test.ts`. Do you think you can fix that?

Hi @adrianmcli, is it still an issue?

**Comment by adrianmcli** [comment]
> > @marcelomorgado looks like there's a conflict w/ `kyber.test.ts`. Do you think you can fix that?
> 
> Hi @adrianmcli, is it still an issue?

Nope, I fixed it!

**Pull request merged**